libs/platform/experience/src/services/layout/plugins/properties/divider/divider-layout.plugin.ts

```diff
@@ -8,9 +8,9 @@ export class DividerLayoutPlugin implements LayoutPlugin {
     return ssrAwaiter(import('./divider.styles').then((m) => m.styles));
```

Comment:
What about renaming the styles `dividerStyles` and export the styles in `index.ts`?

Reply:
if we do this, we should do it everywhere